pgn_downloader/main.py

```diff
@@ -8,17 +8,29 @@ def cli_arguments(args=None):
     """Parse CLI arguments"""
     parser = argparse.ArgumentParser()
     parser.add_argument("username")
+    parser.add_argument("color", help="Color (white/black)")
     parser.add_argument("-o", "--output", default=None, help="Output filename")
 
     args = parser.parse_args()
     args.output = args.output if args.output is not None else f"{args.username}.pgn"
+    
+    if args.color in ["w", "W", "White", "white"]:
```

Comment:
Would be faster to just check the first letter, like`args.color[0].lower() == "w"`.
But better solution in my opinion is too force the argument using `choices`, see comment on line 11